tests/bootstrap.php

```diff
@@ -30,6 +30,9 @@
 		'encoding' => 'UTF-8']);
 Cake\Core\Configure::write('debug', true);
 
+// Disable deprecation warnings, to prevent errors from email setup.
+error_reporting(E_ALL & ~E_USER_DEPRECATED);
```

Comment:
what kind of errors?
this should not be necessary, otherwise PR looks good.

Reply:
Ah, you're right. I added this to stop it throwing "Email::setConfigTransport() is deprecated. Use TransportFactory::setConfig() instead." errors, but then found that I also needed the `phpunit.xml.dist` changes to stop some "View::$response is protected now." messages. Seems that the `phpunit.xml.dist` change alone takes care of both sets, so I'll eliminate this bit.